examples/parol-app/app.rs

```diff
@@ -15,14 +15,10 @@ fn main() {
 
     let mut json_grammar = grammar::Grammar::new();
     match parser::parse(&src, &path, &mut json_grammar) {
-        Ok(json) => {
+        Ok(_) => {
             #[cfg(debug_assertions)]
             {
-                println!("{:#?}", json);
-            }
-            #[cfg(not(debug_assertions))]
-            {
-                std::hint::black_box(json);
+                println!("{}", json_grammar);
```

Comment:
Why are are discarding the parsed result?  The intention here is
- To make sure the optimizer doesn't discard the relevant logic
- To let people see the parse result to check the quality of the parse result

Reply:
No, no. The parse result is in the json_grammar variable. The parse tree is optional.